website/docs/docs/deploy/webhooks.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Webhooks for your jobs"
+description: "Get notifications about your dbt jobs with webhooks."
+---
+
+With dbt Cloud, you can create outbound webhooks to send events (notifications) about your dbt jobs to your other systems. Your other systems can listen for these events to further automate your workflows or to help trigger automation flows you have set up.
+
+A webhook is an HTTP-based callback function that allows event-driven communication between two different web applications. This allows you to get the latest information on your dbt jobs in real time. Without it, you would need to make API calls repeatedly to check if there are any updates that you need to account for (polling). Because of this, webhooks are also called _push APIs_ or _reverse APIs_ and often used for infrastructure development.
```

Comment:
```suggestion
A webhook is an HTTP-based callback function that allows event-driven communication between two different web applications. This allows you to get the latest information on your dbt jobs in real time. Without it, you would need to make API calls repeatedly to check if there are any updates that you need to account for (polling). Because of this, webhooks are also called _push APIs_ or _reverse APIs_ and are often used for infrastructure development.
```